src/lib/constants.ts

```diff
@@ -42,7 +45,7 @@ export function IS_TEST_USER(handle?: string) {
 }
 
 export function IS_PROD_SERVICE(url?: string) {
-  return url && url !== STAGING_SERVICE && url !== LOCAL_DEV_SERVICE
+  return url && url !== STAGING_SERVICE && !url.startsWith(LOCAL_DEV_SERVICE)
```

Comment:
During account creation, this is fine, but once normalized via `URL.toString` it contains a slash so the exact match was failing.